README.md

```diff
@@ -1,3 +1,21 @@
 # TypeScript definitions of MediaWiki front end code interfaces
 
 NOTE: This exists as a separate repository for the time being. It has been created with idea in mind that it those definitions could be maintained together with MediaWiki front end code itself.
+
+## Development
```

Comment:
Very, very minor (feel free to ignore): I'm not sure if it makes sense but we may want to encourage users to use this ["fresh-node" project](https://github.com/wikimedia/fresh#usage) and omit all of the Docker code. It encapsulates some of the user / group workarounds. Or maybe we need the Dockerfiles for the GitHub workflows?

Reply:
Docker files are not needed for the Github workflows I think. Using fresh-node might make sense here. Will try to replace.